src/system/applications/combat/combat-tracker.ts

```diff
@@ -0,0 +1,151 @@
+import CosmereCombatant from './combatant';
+
+export default class CosmereCombatTracker extends CombatTracker {
+    //overrides default tracker template to implement slow/fast buckets and activation button.
+    //eslint-disable-next-line -- eslint rules want readonly field
+    get template() {
+        return 'systems/cosmere-rpg/templates/combat/combat-tracker.hbs';
+    }
+
+    //modifies data being sent to the combat tracker template to add turn speed, type and activation status and splitting turns between the initiative phases.
+    async getData(
+        options?: Partial<ApplicationOptions> | undefined,
+    ): Promise<object> {
+        const data = (await super.getData(options)) as {
+            turns: CosmereTurn[];
+            [x: string]: unknown;
+        };
+        data.turns = data.turns.map((turn) => {
```

Comment:
Is there a reason to type this as `[x: string]: unknown`? 
Couldn't we type the whole thing as:
```ts
turns: CosmereTurn[];
fastPlayers: CosmereTurn[];
slowPlayers: CosmereTurn[];
fastNPC: CosmereTurn[];
slowNPC: CosmereTurn[];
```

Keeping everything properly typed will make it easier for other people to make changes to this code should it be needed in the future.

Reply:
I had it unknown for something I was testing when trying to set it up, fixed